include/nuttx/lcd/lcd.h

```diff
@@ -90,7 +92,8 @@ struct lcd_planeinfo_s
 
   int (*putarea)(FAR struct lcd_dev_s *dev, fb_coord_t row_start,
                  fb_coord_t row_end, fb_coord_t col_start,
-                 fb_coord_t col_end, FAR const uint8_t *buffer);
+                 fb_coord_t col_end, FAR const uint8_t *buffer,
+                 fb_coord_t stride);
```

Comment:
let add the same parameter to get_area too?

Reply:
Yes, it makes total sense! I'll update the PR, thanks!

Reply:
Updated it ;)